src/io/flutter/logging/FlutterLogEntry.java

```diff
@@ -39,7 +39,7 @@
   private String message;
   // TODO(pq): consider making data an Instance or JsonElement
```

Comment:
remove this TODO and document what types data is allowed to have.

Reply:
Good idea. Done